robot/Cumulus/resources/NPSP.py

```diff
@@ -1223,6 +1223,31 @@ def select_value_from_dropdown(self,dropdown,value):
         self.selenium.get_webelement(locator).click()
         self.wait_for_locator('popup')
         self.npsp.click_link_with_text(value)
+
+    @capture_screenshot_on_error
+    def click_rd2_modal_button(self, name):
+        """Based on the button name (Cancel)  or (Save) on the modal footer, select and click on the respective button"""
+        btnlocator = npsp_lex_locators["button-with-text"].format(name)
+        self.selenium.scroll_element_into_view(btnlocator)
+        ele = self.selenium.get_webelement(btnlocator)
```

Comment:
no need of this line, u can directly use click element

Reply:
changed